File: core/src/main/java/org/mifos/mobilewallet/core/domain/model/NewAccount.java
package org.mifos.mobilewallet.core.domain.model;

import java.util.Date;

/**
 * Created by ankur on 25/June/2018
 */

public class NewAccount {

    private int clientId;
    private String productId;
    private Date submittedOnDate;
    private String accountNo;
    private String locale;
    private String dateFormat;

    public NewAccount(int clientId, String accountNo) {
        this.clientId = clientId;
        this.accountNo = accountNo;
    }
}

File: core/src/main/java/org/mifos/mobilewallet/core/domain/model/NewAccount.kt
package org.mifos.mobilewallet.core.domain.model

import java.util.*

/**
 * Created by ankur on 25/June/2018
 */
data class NewAccount(
        var clientId: Int? = null,
        var accountNo: String? = null) {

    private val productId: String? = null
    private val submittedOnDate: Date? = null
    private val locale: String? = null
    private val dateFormat: String? = null

}